projects/demo/src/modules/app/search/index.html

```diff
@@ -0,0 +1 @@
+<div id="docsearch"></div>
```

Comment:
Could we attach this `id` directly on `<tui-algolia-search />` and remove this template-file ?